build.gradle

```diff
@@ -46,12 +46,12 @@ plugins {
 
 allprojects {
   repositories {
-    artifactoryRelease()
-    artifactorySnapshot()
-    mobileLibs()
+//    artifactoryRelease()
+//    artifactorySnapshot()
```

Comment:
We can remove this and `artifactoryRelease`

Reply:
Got it. Thanks. Will remove in the upcoming one.